addons/website/models/website.py

```diff
@@ -527,9 +527,9 @@ def viewref(self, view_id, raise_if_not_found=True):
             if views:
                 view = views.filter_duplicate()
             else:
-                view = self.env.ref(view_id)
+                view = self.env.ref(view_id, raise_if_not_found=raise_if_not_found)
```

Comment:
It doesn't actually matter if it's `raise_if_not_found=False` or `raise_if_not_found=raise_if_not_found` because if the view is empty, there is another `if raise_if_not_found` later on the method.

We just need to decide if we want in that case to raise the original exception from ref or the new exception from viewref.

What is definitely not correct is to pass True (default value) like before.